# cam_intrinsics: [ 
#   500.32414335, 311.14957496, 500.40247593, 261.95543036
# ]

cam_intrinsics: [ 
  628.8233032226562, 646.732666015625, 628.3672485351562, 364.4508361816406
]


tag_size: 0.3